feat(curriculum): add linked list workshop (#61424)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-linked-list-class/6888f9827637322d4f309192.md

---
id: 6888f9827637322d4f309192
title: Step 18
challengeType: 20
dashedName: step-18
---

# --description--

> Next, create a variable called `current_node` inside the `remove` method. It should have a value of `self.head`.

Reviewer: ```suggestion
Next, create a variable called `current_node` inside the `remove` method and assign it the value of `self.head`.
```

# --hints--

> `current_node` should have a value of `self.head`.

Reviewer: ```suggestion
You should create a variable `current_node` and assign it the value of `self.head` inside the `remove` method.
```
